crates/bevy_ui/src/ui_node.rs

```diff
@@ -2266,6 +2266,39 @@ impl ResolvedBorderRadius {
     };
 }
 
+#[derive(Component, Copy, Clone, Debug, PartialEq, Reflect)]
+#[reflect(Component, PartialEq, Default)]
+#[cfg_attr(
+    feature = "serialize",
+    derive(serde::Serialize, serde::Deserialize),
+    reflect(Serialize, Deserialize)
+)]
+pub struct BoxShadow {
+    /// The shadow's color
+    pub color: Color,
+    /// Offset increasing to the right and down
+    pub x_offset: Val,
+    pub y_offset: Val,
+    /// Difference in size from occluding uninode
+    pub x_spread: Val,
```

Comment:
Hmm, I don't like this field name either. It's fine though and I can't think of anything better.